pkg/rc/farm.go

```diff
@@ -89,7 +88,7 @@ func NewFarm(
 // Set the metrics registry to use (see
 // https://godoc.org/github.com/rcrowley/go-metrics#Registry). If set,
 // performance metrics will be recorded to the registry.
-func (rcf *Farm) SetMetricsRegistry(registry metrics.Registry) error {
+func (rcf *Farm) SetMetricsRegistry(registry rcmetrics.MetricsRegistry) error {
```

Comment:
what's the benefit of that? That would also work but i guess i like the logging in the implementation of `RecordRCProcessingTime`, and we would probably mutate the `rcmetrics.Metrics` instance passed in to layer on metrics